app/frontend/components/shared/permit-applications/requirement-form.tsx

```diff
@@ -212,7 +270,6 @@ export const RequirementForm = observer(
               </Link>
             </Text>
           </Box>
-          {showHelpButton && <BuilderTopFloatingButtons mr={0} mt={0} />}
```

Comment:
Just clarifying if the help sidebar button is added somewhere else or if this is intentionally  removed.

Reply:
Help button has a new home in, its been moved to `builder-top-floating-buttons`